install/configuring_inventory_file.adoc

```diff
@@ -217,6 +217,13 @@ xref:configuring-node-host-labels[Configuring Node Host Labels] for details.
 |This variable enables the template service broker by specifying one or more
 namespaces whose templates will be served by the broker.
 
+|`openshift_master_bootstrap_auto_approve`
+|This variable enables TLS bootstrapping auto approval, which allows nodes to
+automatically join the cluster when provided a bootstrap credential. Set to
+`true` if you will enable the 
```

Comment:
s/if you will enable/you enable ?

Reply:
Normally, yes, but I'm afraid that "you enable" might imply that you can enable the auto-scaler **now**, which you can't.